m.zsh

```diff
@@ -13,7 +23,7 @@ function m() {
 function ml() {
   # this if statement is needed for testing
   # as fc doesn't work in zunit
```

Comment:
This comment is unclear to me: what is `fc`?

Reply:
It's used below (https://github.com/samhstn/m/pull/18/files/b8ddfa4ec3853b54965b7f7e0a98b39cd91a52c6#diff-52508c3469506348ec7d7418b80758d5R29), I'll add a comment to clarify though and reference the docs (perhaps here: https://github.com/samhstn/m/pull/18/files/b8ddfa4ec3853b54965b7f7e0a98b39cd91a52c6#diff-52508c3469506348ec7d7418b80758d5R29)